packages/node_modules/@ciscospark/widget-base/README.md

```diff
@@ -0,0 +1,31 @@
+# Spark Space Widget _(@ciscospark/widget-base)_
+
+* THIS WIDGET CONTAINS EXPERIMENTAL CODE *
+
+> This base React component does the initial React, Spark authentication, and Redux setup for all Spark widgets.
+
+## Usage
+
+### Quick Start
+
+To automatically connect reducers into a Redux store, setup Cisco Spark connections, and register your widget into the global browser store, you can call the `constructSparkWidget` method like so:
+
+``` js
+import constructSparkWidget from `@ciscospark/widget-base`;
+
+function MyWidget(props) {
+  return <div>My Widget</div>;
+}
+
+const MyWidgetWithSpark = constructSparkWidget(`nameOfMyWidget`, Widget, {
+  intl: {
```

Comment:
Long-term, I'm not sure this is how we want to expose intl, but let's talk about it at some point. no need to hold this PR on it.